components/sup/src/manager/mod.rs

```diff
@@ -1741,6 +1779,23 @@ impl Manager {
     }
 }
 
+fn tls_config(tls_files: (PathBuf, PathBuf)) -> Result<ServerConfig> {
+    let (kp, cp) = tls_files;
+    let mut config = ServerConfig::new(NoClientAuth::new());
+    let key_file = &mut BufReader::new(File::open(&kp).unwrap());
+    let cert_file = &mut BufReader::new(File::open(&cp).unwrap());
```

Comment:
These unwraps would be better as `?`, I think.

Reply:
Yep